deploy/docs/Best_Practices.md

```diff
@@ -784,3 +784,63 @@ metadata:
       - name: KUBERNETES_SERVICE_PORT
         value: '12345'
 ```
+
+## OpenTelemetry queueing and batching
+
+OpenTelemetry goes with several parameters related to queue management.
+
+For [batch processor][batch_processor]:
+
+- `send_batch_size` defines the number of items (logs, metrics, traces) in one batch before it's sent further down the pipeline.
+- `timeout` defines time after which the batch is send regardless of the size (can be lower than `send_batch_size`).
+- `send_batch_max_size` is an upper limit of the batch size.
+
+_We could say that `send_batch_size` is soft limit and `send_batch_max_size` is hard limit of the batch size._
+
+For [sumologic exporter][sumologic_exporter]:
+
+- `max_request_body_size` defines maximum size of requests to sumologic before compression.
+- `timeout` defines connection timeout. It is recommended to adjust this value in relation to `max_request_body_size`.
+
+- `sending_queue.num_consumers` is the number of consumers that dequeue batches. It translates to maximum number of parallel connections to the sumologic backend.
+- `sending_queue.queue_size` is capacity of the queue in terms of batches (batches can vary between `1` and `send_batch_max_size`)
+
+**As effective value of `sending_queue.queue_size` depends on current traffic,**
+**there is no way to figure out optimal PVC size in relation to `sending_queue.queue_size`.**
+**Due to that, we recommend to set `sending_queue.queue_size` to high value in order to use maximum resources of PVC.**
```

Comment:
We should make clear what is going to happen if the PVC is filled.

Reply:
added below